test/test_torch.py

```diff
@@ -13436,7 +13432,7 @@ def test_addcdiv(self, device):
         def _test_addcdiv(a, alpha, b, c):
             actual = torch.addcdiv(a, b, c, value=alpha)
             # implementation of addcdiv downcasts alpha. arithmetic ops don't.
-            if not actual.dtype.is_floating_point:
+            if not (actual.dtype.is_floating_point or actual.dtype.is_complex):
                 alpha = int(alpha)
```

Comment:
Github is having a hard time loading test_torch. However, the int(alpha) call looks a bit fishy to me, because the helper function calls _test_addcdiv with 0.5. `int(0.5)` resolves to 0, so the choice of 0.5 is not the best thing for testing integer dtypes with addcdiv. But this is a preexisting problem.
```
def _helper():
    _test_addcdiv(
        non_zero_rand((2, 2), dtype=dtype, device=device),
        0.5,
        non_zero_rand((2, 2), dtype=dtype, device=device),
        non_zero_rand((2, 2), dtype=dtype, device=device))
```

Reply:
agreed updated!